purchase_order_product_recommendation/__manifest__.py

```diff
@@ -11,9 +11,10 @@
     "license": "AGPL-3",
     "application": False,
     "installable": True,
-    "depends": ["purchase_stock"],
+    "depends": ["purchase_stock", "report_xlsx"],
```

Comment:
Isn't this adding a very off the main target dependency? Why not an extra module?

Reply:
Ok so, I will preserve this PR for the fix and do a new one for the XLS export module